app/models/heron/factories/sample.rb

```diff
@@ -105,7 +105,7 @@ def params_for_sample_creation
       end
 
       def params_for_aliquot_creation
-        { sample: sample, study: study }.merge(@params.dig(:aliquot) || {})
+        { sample: sample, study: study }.merge(@params[:aliquot] || {})
```

Comment:
These Heron changes are a result of rubocop no longer ignoring the file (because of its destination changes). I believe `dig(:key)` and `[:key]` both return nil if they don't exist. See [rubocop's reasoning here](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/SingleArgumentDig).